query_modules/schema.cpp

```diff
@@ -108,42 +109,33 @@ void Schema::ProcessPropertiesRel(mgp::Record &record, const std::string_view &t
   record.Insert(std::string(kReturnMandatory).c_str(), mandatory);
 }
 
-struct Property {
-  std::string name;
-  mgp::Value value;
+struct PropertyInfo {
+  std::unordered_set<std::string> values;
+  int64_t number_of_property_occurrences = 0;
```

Comment:
Is this number of property tuple occurances?

Reply:
This counter is used to check how many times a property has appeared with a certain label. If the number of occurrences of given property is the same as number of occurrences for a certain label then all labels with that name have this property, hence the property is mandatory.